wasm-and-containers/README.md

```diff
@@ -5,13 +5,15 @@ This workshops contains:
   - Includes using Spin, building Docker containers, and using Kubernetes
 
 **Last Updated**
-This workshop was last updated for KubeCon North America 2023
+This workshop was last updated for Civo Navigate Austin, 2024.
 
 ## Workshop Modules
 
 - [0. Setup](./workshop/00-setup.md)
 - [1. Getting started with Spin and WebAssembly](./workshop/01-spin-getting-started.md)
 - [2. Run your first WebAssembly application in a container](./workshop/02-running-in-a-container.md)
```

Comment:
I believe the container section will need to be updated to use the `v2` shim:
`docker run -d --platform=wasi/wasm --runtime=io.containerd.spin.v2 -p 3000:80`

Reply:
@jpflueger just fixed it. Thanks!